frontend/src/components/StatusPage/StatusPageDetailData.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { formatDistanceToNow, parseISO } from "date-fns";
+import { Address, Status, Part } from "solo-types";
+
+interface StatusPageDetailDataProps {
+  address: Address[];
+  status: Status[];
+  part: Part[];
+}
+
+const StatusPageDetailData: React.FC<StatusPageDetailDataProps> = ({
+  address,
+  status,
+  part
+}) => {
+  const shipperAddy = address.find(addy => addy.address_type === "2")?.name;
+  const receiverAddy = address.find(addy => addy.address_type === "3")?.name;
+  const quantity = status[0]?.qty || 0;
```

Comment:
Just want to check my own assumptions. Is there a real benefit or difference between `|| 0` and `?? 0`

Reply:
Added explicit conversion step from the structure of the api response to a more frontend focused data structure.

Reply:
No longer necessary either way